tests/sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_costs_v1/monthly_costs_v1.schema.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
These optional `*.schema.json` files for tests are intended to specify the schemas of _input_ tables, not the output table.